javascript/node/selenium-webdriver/firefox/index.js

```diff
@@ -367,14 +373,9 @@ class Driver extends webdriver.WebDriver {
       let service = createGeckoDriverService(binary);
       serverUrl = service.start();
       onQuit = () => service.kill();
+      caps.set(Capability.PROFILE, profile.encode());
```

Comment:
I have seen profiles that are in the hundreds of megabyte in size, which might matter if you’re running tests, say, in a disposable Linux container with limited disk storage.

On the other hand it would add more complexity to the implementation, so it should be well-weighed decision against that.